src/useCapabilitiesSafe.ts

```diff
@@ -0,0 +1,36 @@
+import type { Chain } from 'viem';
+import { useAccount } from 'wagmi';
+import { useCapabilities } from 'wagmi/experimental';
+import type { WalletCapabilities } from './types';
+
+export function useCapabilitiesSafe({
+  chain,
```

Comment:
Going forward we are moving away from using `chain`, instead we will have directly `chainId`.

Reply:
changed in latest diff